.azure-pipelines.yml

```diff
@@ -94,130 +94,57 @@ jobs:
-
-    - bash: |
-        echo "##vso[task.setvariable variable=tagName;]`echo $BUILD_SOURCEBRANCH | sed -e 's|refs/tags/||'`"
-      displayName: Set tag name
-      condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
-    - bash: |
-        echo "##vso[task.setvariable variable=tagName;]dev"
-      displayName: Set tag name to "dev"
-      condition: not(startsWith(variables['Build.SourceBranch'], 'refs/tags/'))
-
-    - bash: echo "##vso[task.setvariable variable=basename;]wasmtime-$(tagName)-x86_64-windows"
-      displayName: Configure basename var
-      condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))
-    - bash: echo "##vso[task.setvariable variable=basename;]wasmtime-$(tagName)-x86_64-apple"
-      displayName: Configure basename var
-      condition: and(succeeded(), eq(variables['Agent.OS'], 'Darwin'))
-    - bash: echo "##vso[task.setvariable variable=basename;]wasmtime-$(tagName)-x86_64-linux"
-      displayName: Configure basename var
-      condition: and(succeeded(), eq( variables['Agent.OS'], 'Linux' ))
-
-    - bash: |
-        set -e
-        mkdir -p $BUILD_BINARIESDIRECTORY/$BASENAME
-        if [ "$AGENT_OS" = "Windows_NT" ]; then
-          ext=.exe
-        fi
-        cp LICENSE README.md target/release/{wasmtime,wasm2obj}$ext $BUILD_BINARIESDIRECTORY/$BASENAME
-      displayName: Copy binaries
-
-    - bash: |
-        set -e
-        export WT_VERSION=`cat Cargo.toml | sed -n 's/^version = "\([^"]*\)".*/\1/p'`
-        "$WIX/bin/candle" -arch x64 -out target/wasmtime.wixobj installer/msi/wasmtime.wxs
-        "$WIX/bin/light" -out $BUILD_ARTIFACTSTAGINGDIRECTORY/$(basename).msi target/wasmtime.wixobj -ext WixUtilExtension
-        rm $BUILD_ARTIFACTSTAGINGDIRECTORY/$(basename).wixpdb
-      displayName: Create installer (Windows)
-      condition: eq(variables['Agent.OS'], 'Windows_NT')
-
-    - task: ArchiveFiles@2
-      inputs:
-        rootFolderOrFile: $(Build.BinariesDirectory)/$(basename)
-        archiveType: 'zip'
-        archiveFile: '$(Build.ArtifactStagingDirectory)/$(basename).zip'
-      displayName: Archive files (Win)
-      condition: and(succeeded(), eq(variables['Agent.OS'], 'Windows_NT'))
-    - task: ArchiveFiles@2
-      inputs:
-        rootFolderOrFile: $(Build.BinariesDirectory)/$(basename)
-        archiveType: 'tar'
-        tarCompression: 'xz'
-        archiveFile: '$(Build.ArtifactStagingDirectory)/$(basename).tar.xz'
-      displayName: Archive files (Unix)
-      condition: and(succeeded(), ne(variables['Agent.OS'], 'Windows_NT'))
-    - task: PublishPipelineArtifact@1
-      inputs:
-        path: $(Build.ArtifactStagingDirectory)/
-        artifactName: 'bundle-$(Agent.OS)'
+# Build the Linux release binary in an older Linux container (in this case
+# Centos 6)
+- job: Build_linux
+  variables:
+    toolchain: stable
+  container:
+    image: centos:6
+    options: "--name ci-container -v /usr/bin/docker:/tmp/docker:ro"
+  steps:
```

Comment:
Should these perhaps also be moved into a template?

Reply:
Oh I figured that the first steps here wouldn't want to be in the template since they're specific to just this one container, so they'd be skipped on OSX/Windows anyway.

Reply:
Fair point, let's leave it as-is.